projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -2701,14 +2705,21 @@ t_server
 
 t_address
 :
+   ADDRESS
    (
-      ADDRESS ((IPV4 IP_ADDRESS) | (IPV6 IPV6_ADDRESS)) NEWLINE
-   )
+      (
+         IPV4 IP_ADDRESS
+      )
+      |
+      (
+         IPV6 IPV6_ADDRESS
+      )
+   ) NEWLINE
 ;
 
 t_key
 :
-    KEY DIGIT HEX_DIGIT+
+   KEY DEC VARIABLE NEWLINE
```

Comment:
Will add the tests, but after checking further, the syntax allows non-hex strings (when DEC is omitted or 0). It is an encrypted string from a rotational cipher (does not have to be hexadecimal). Will update the rule accordingly.